data/ecosystems/c/celo.toml

```diff
@@ -110,6 +113,7 @@ sub_ecosystems = [
   "Seam Money",
   "simplefi",
   "Spirals Protocol",
+  "Squid",
```

Comment:
how's Squid using Celo infra? I see it uses Axelar and [can't find references to Celo in their docs](https://docs.squidrouter.com/?q=celo)

Reply:
We work pretty closely with them. Squid supports Celo both for interchain routing but as of recently also intrachain on Celo routing. They have also implemented support for minting cUSD, cEUR, and cREAL directly via their router.